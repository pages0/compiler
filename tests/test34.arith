let x : <bool> = ref false in
let y : <bool> = ref !x   in
x := !y or true;
if !y then !x else true