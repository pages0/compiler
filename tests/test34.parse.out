( let [ x ( ref false ) ] ( let [ y ( ref ( ! x ) ) ] ( ; ( != x ( or ( ! y ) true ) ) ( if ( ! y ) ( ! x ) true ) ) ) )